### PR TITLE
adds footer and tos page

### PR DESCRIPTION
**Pull request by georgeweiler**

### CODE REVIEW

File: src/components/Footer.tsx [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+          </Link>
+          {/* TODO: add this link when page goes live */}
+          {/* <Link external to="http://activate.codefi.network/eth2"> */}
+          {/*  Calculator */}
+          {/* </Link> */}
+          <Link
+            external
+            to="https://github.com/runtimeverification/deposit-contract-verification/blob/96434de/deposit-contract-verification.pdf"
+          >
+            Deposit Verification
```
Comment (hwwhww) [inline src/components/Footer.tsx:105]: "Deposit Verification" may look like a deposit data verifier. Maybe use something like "Formal Verification Report"?

File: src/pages/TermsOfService/index.tsx [hunk 2]
```diff
@@ -0,0 +1,557 @@
...
+                our{' '}
+                <Link
+                  primary
+                  external
+                  inline
+                  to="https://ethereum.org/en/cookie-policy/"
+                >
+                  Cookie Policy
+                </Link>
+                ,incorporated herein by reference.
```
Comment (CarlBeek) [inline src/pages/TermsOfService/index.tsx:108]:
```suggestion
                , incorporated herein by reference.
```

File: src/components/Footer.tsx (hunk 1 above)
Comment (CarlBeek) [reply src/components/Footer.tsx:105]: I agree with @hwwhww here "Formal Verification Report" is more accurate.
```suggestion
            Formal Verification Report
```

**Review by CarlBeek — APPROVED** [review]
LGTM!

**Review by hwwhww — APPROVED** [review]
lgtm

**Comment by hwwhww** [comment]
@georgeweiler I'm merging it now for getting a preview on `dev` and `master`.

**Pull request merged**